src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -2916,7 +2916,7 @@ private class ConsoleHostStartupException : Exception
             {
             }
 
-            protected
+            internal
```

Comment:
Please revert unrelated change.

Reply:
Fixed in a573ebf. Protected member cannot exist in sealed class, see [CS0628](https://docs.microsoft.com/dotnet/csharp/misc/cs0628)